akka-http-core/src/main/scala/akka/http/scaladsl/Http.scala

```diff
@@ -1108,8 +1109,26 @@ object Http extends ExtensionId[HttpExt] with ExtensionIdProvider {
 
   def lookup() = Http
 
-  def createExtension(system: ExtendedActorSystem): HttpExt =
+  // Used for ManifestInfo.checkSameVersion
+  private def allModules: List[String] = List(
+    "akka-parsing",
+    "akka-http-core",
+    "akka-http2-support",
+    "akka-http",
+    "akka-http-caching",
+    "akka-http-testkit",
+    "akka-http-tests",
+    "akka-http-marshallers-scala",
+    "akka-http-marshallers-java",
+    "akka-http-spray-json",
+    "akka-http-xml",
+    "akka-http-jackson"
+  )
+
+  def createExtension(system: ExtendedActorSystem): HttpExt = {
+    ManifestInfo(system).checkSameVersion("Akka HTTP", allModules, logWarning = true)
```

Comment:
There are also checks above at 

https://github.com/akka/akka-http/blob/3560e98754e7bf0d684a9b316926737dfa04c6a8/akka-http-core/src/main/scala/akka/http/scaladsl/Http.scala#L57-L58

Could you change it so that all the checks are done in one place?

Reply:
I moved the check to HttpExt, after the other version checks

Reply:
Thanks.